<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrieren BWS - Wettsystem</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Registrieren</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegistrieren">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrieren</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>127</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblBenutzername">
      <property name="text">
       <string>Benutzername:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtBenutzername"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>127</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="lblEmail">
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtEmail"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>127</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="lblBenutzername_2">
      <property name="text">
       <string>Vorname:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtVorname">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>127</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="lblBenutzername_4">
      <property name="text">
       <string>Klasse:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtKlasse">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>127</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="lblBenutzername_3">
      <property name="text">
       <string>Nachname:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtNachname">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>127</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="lblBenutzername_5">
      <property name="text">
       <string>Passwort:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtPasswort">
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>127</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="lblBenutzername_6">
      <property name="text">
       <string>Passwort wiederholen:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtPasswortWiederholt">
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="bwswettsystemlogo.qrc">:/logo/0001.ico</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edtBenutzername</tabstop>
  <tabstop>edtVorname</tabstop>
  <tabstop>edtNachname</tabstop>
  <tabstop>edtEmail</tabstop>
  <tabstop>edtKlasse</tabstop>
  <tabstop>btnRegistrieren</tabstop>
 </tabstops>
 <resources>
  <include location="bwswettsystemlogo.qrc"/>
 </resources>
 <connections/>
</ui>
